assets/css/header-navbar.css

```diff
@@ -0,0 +1,153 @@
+body{
+    position: relative;
+}
+main{
+    background-color: #f9f7ff;
+}
+header{
+    height:96.5vh;
+    border-radius: 38px;
+    width:90%;
+    margin:0 auto;
+    box-sizing: border-box;
+    padding-top: 1rem !important;
+    position: relative;
+}
+.navbar{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items:center;
+    width: 100%;
+    padding: 0.5rem;
+    padding-left: 3.3rem;
+    box-shadow: 0 3px 9px 5px #dddddd73;
+    border-radius: 38px;
+    margin-top: 1rem;
+    background-color: #fff;
+}
+.navbar__logo{
+    color: var(--main-color);
+    text-transform: uppercase;
+    padding: 3px 6px;
+    border-radius: 3px;
+    border: solid 2px var(--main-color);
+    background-color: #ffffff;
+    line-height: 1;
+}
+.navbar__logo > h2{
+    font-family: 'Rubik' ,sans-serif;
+    font-weight: 100;
+    font-size: 2.2rem;
+}
+.navbar__list{
+    margin-left: auto;
+    margin-right: 2.6rem;
+}
+.navbar__list > li{
+    display: inline-block;
+}
+.navbar__link{
+    color: #7a8baa;
+    text-transform: uppercase;
+    padding: 0.7rem 0.8rem;
+    letter-spacing: .9px;
+    font-size: 13px;
+}
+.navbar__link:hover{
+    transition:.6s;
+}
+.navbar__link--active{
+    color: var(--main-color) !important;
+}
+.navbar__button{
```

Comment:
can page have more navbar so that using name more clear

Reply:
I didn't understand you these classes are used for any links on navbar